server/server.js

```diff
@@ -1,23 +1,29 @@
 const express = require('express');
-let app = express();
-let { retrieve, seeder } = require('./db/index.js')
+const app = express();
+const { retrieve } = require('./db/index.js')
+const cors = require('cors')
+const bodyParser = require ('body-parser');
+require('dotenv').config()
 
 app.use(express.static(__dirname + '/../client/dist'));
+app.use(cors())
 
-app.get('/', function (req, res) {
-  console.log('Got request');
-  seeder();
-  res.end();
+app.get('/amenities-api/amenities', function (req, res) {
+  retrieve({})
+    .then(data => {
+      if (data.length === 0) { res.status(400).json("ID not found") }
```

Comment:
Error message is misleading here, could respond with "No records found" or something along those lines